x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/UpdateConnectorFilteringAction.java

```diff
@@ -96,26 +97,30 @@ public ActionRequestValidationException validate() {
                 validationException = addValidationError("[connector_id] cannot be [null] or [\"\"].", validationException);
             }
 
-            // If [filtering] is not present in the request payload it means that the user should define both [rules] and [advances_snippet]
+            // If [filtering] is not present in the request payload it means that the user should define [rules] and/or [advances_snippet]
             if (filtering == null) {
                 if (rules == null && advancedSnippet == null) {
+                    validationException = addValidationError("[advanced_snippet] and [rules] cannot be both [null].", validationException);
```

Comment:
We want to discourage users from modifying `filtering` directly, hence I just mention advanced_snippet and rules here